src/test/java/posters/pageobjects/pages/user/PersonalDataPage.java

```diff
@@ -24,9 +24,10 @@
 
     @Override
     @Step("ensure this is a personal data page")
-    public void isExpectedPage()
+    public PersonalDataPage isExpectedPage()
     {
         headline.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.